packages/playground/src/index.css

```
@@ -76,6 +76,15 @@ html::-webkit-scrollbar-thumb:hover {
   }
 }
 
+@keyframes shimmer-text {
+  0% {
+    background-position: 200% 0;
+  }
+  100% {
+    background-position: -200% 0;
+  }
```

Stylelint (17.12.0)

[error] Expected empty line before rule (rule-empty-line-before)

(rule-empty-line-before)